app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Update Review</h1>
+
+<%= render :partial => "form", locals: {
+                                        path: "/shelters/#{@review.shelter_id}/reviews/#{@review.id}",
```

Comment:
We had to change this from /shelters/shelter_id to this current path because the patch was not routing to the correct place